plugin/providers/phpipam/subnet_structure.go

```diff
@@ -138,6 +138,8 @@ func resourceFirstFreeSubnetSchema() map[string]*schema.Schema {
 	case k == "parent_subnet_id" || k == "subnet_mask" :
 			v.Required = true
 			v.ForceNew = true
+		case k == "section_id":
+			v.Required = true
```

Comment:
It is possible to change required to optional?
A lot of users users currently use resource creation without this parameter. If it will be mandatory - It will break backwards compatibility

Suggestion:
```v.Optional = true```